### DISCUSSION TITLE
[Blog Post] Credential leak hunting using h8mail and the new COMB (Combination Of Many Breaches)

### DISCUSSION THREAD

**Discussion by khast3x** [khast3x/h8mail#117] [Publications]
[Blog post link](https://khast3x.club/posts/2021-02-17-h8mail-with-COMB/).

**Comment by davemclarenflat** [comment]
Can anyone confirm if the COMB includes the data from the original Breach Compilation? As in, would somebody so inclined as to have that kind of thing keep both the COMB data and original Breach Compilation data and scan both when searching with h8mail, or can just the COMB be scanned to achieve the same result.

Thanks in advance.
